assets/components/directDebit/directDebitActions.js

```diff
@@ -78,6 +81,7 @@ function payDirectDebitClicked(callback: Function): Function {
           throw new Error('incorrect_input');
         }
         callback(undefined, bankAccountNumber, bankSortCode, accountHolderName);
+        dispatch(closeDirectDebitPopUp());
```

Comment:
Maybe a UX detail here. Why are we closing the DirectDebitPopUpForm here? 
If everything is OK the user will be redirected to the thank you page, if something is wrong I think we would like to keep the user in the same form to introduce the correct input.

Reply:
We validate all that we can within the form, this happens after we send the request off to the step functions. The issue then is that all the polling and any error message happens in the page behind the pop up so if we don't close the pop up the error message is hidden.